pymc3/tests/test_initvals.py

```diff
@@ -47,3 +51,15 @@ def test_automatically_assigned_test_values(self):
         rv = pm.HalfFlat.dist()
         assert hasattr(rv.tag, "test_value")
         pass
+
+
+class TestMoment:
+    def test_basic(self):
+        rv = pm.Flat.dist()
+        assert get_moment(rv).eval() == np.zeros(1)
```

Comment:
A random draw from a scalar RV has a `shape == ()`. Shouldn't the moment have the same scalar shape?